src/content/en/2021/caching.md

```diff
@@ -17,4 +19,140 @@ featured_stat_3: TODO
 featured_stat_label_3: TODO
 ---
 
-## TODO
+## Introduction
+
+Over the past decade ...
+
+
+## CDN Cache adoption
+
+Talk about CDN adoptions, 14% of desktop pages use a Cloudflare CDN.
+
+{{ figure_markup(
+  image="to-do.png",
+  caption="Top CDNs adoption",
+  description="TODO",
+  chart_url="TODO",
+  sheets_gid="58739923",
+  sql_file="todo.sql"
+) }}
+
+## Service Worker adoption
+
+Talk about Service Worker adoptions, 1.2% of desktop pages registered a Service Worker, but 9% of mobile pages in the top 1000 registered a SW.
+
+{{ figure_markup(
+  image="to-do.png",
+  caption="Service Worker adoption,
+  description="TODO",
+  chart_url="TODO",
+  sheets_gid="802609299",
+  sql_file="todo.sql"
+) }}
+
+## Headers adoption
+
+Talk about headers trends, Cache-Control vs ETag vs Expires vs Last-Modified.
+
+### Cache Control
+
+Talk about Cache Control adoptions, 74% of responses on mobile pages include the Cache-Control header
+
+{{ figure_markup(
+  caption="The percent of pages that use the Cache-Control header.",
+  content="74%",
+  classes="big-number",
+  sheets_gid="2102749619",
```

Comment:
@rviscomi sheet `2102749619 ` would be possible have a figure showing the "The percent of pages that use the Cache-Control header". Maybe if we have data from other years have a level of comparison.

Reply:
We only have data on the % of requests, not % of pages. Is that ok?

year | desktop | mobile
-- | -- | --
2019 | 72% | 72%
2020 | 74% | 74%
2021 | 75% | 74%

Reply:
Yeah that's perfect! Thank you